.pre-commit-config.yaml

```
@@ -1,41 +1,38 @@
 repos:
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.7.2
```

```suggestion
    rev: v0.8.2
```

Let's use the latest version: https://github.com/astral-sh/ruff/releases/tag/0.8.2